sql/bank_test.go

```diff
@@ -26,16 +26,36 @@ import (
 	"testing"
 
 	"github.com/cockroachdb/cockroach/server"
+	cockroachSql "github.com/cockroachdb/cockroach/sql"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
 var maxTransfer = flag.Int("max-transfer", 999, "Maximum amount to transfer in one transaction.")
 var numAccounts = flag.Int("num-accounts", 999, "Number of accounts.")
 
-// BenchmarkBank mirrors the SQL performed by examples/sql_bank, but structured
-// as a benchmark for easier usage of the Go performance analysis tools like
-// pprof, memprof and trace.
-func BenchmarkBank(b *testing.B) {
+// BenchmarkBankNoDescriptorCache mirrors the SQL performed by examples/sql_bank,
+// but structured as a benchmark for easier usage of the Go performance analysis
+// tools like pprof, memprof and trace.
+// It does not make use of the system cache.
+func BenchmarkBankNoDescriptorCache(b *testing.B) {
+	prev := cockroachSql.TestingDisableDescriptorCache
+	cockroachSql.TestingDisableDescriptorCache = true
+	defer func() { cockroachSql.TestingDisableDescriptorCache = prev }()
```

Comment:
as a pattern, it scales better (e.g. when you need to reset more than one variable). In any case, I'd rather the `defer` immediately follow the `prev :=`